packages/adapter-auto/package.json

```diff
@@ -25,7 +25,7 @@
 		"adapters.js"
 	],
 	"scripts": {
-		"lint": "prettier --check . --config ../../.prettierrc --ignore-path .gitignore",
```

Comment:
I saw these were removed - could someone clarify why? Or why they were needed before? (probably related to the major version bump?)